s4e-web/src/app/views/settings/groups/state/group.service.ts

```diff
@@ -68,4 +68,9 @@ export class GroupService {
         membersEmails: {_: users.members.map(user => user.email)}
       })));
   }
+
+  delete(instSlug: string, groupSlug: string) {
+    return httpDeleteRequest$(this.http, `${this.config.apiPrefixV1}/institutions/${instSlug}/groups/${groupSlug}`, this.store)
```

Comment:
Could you extract `${this.config.apiPrefixV1}/institutions/${instSlug}/groups/${groupSlug}` into cont url?